File: C/Easy/3232.c
bool canAliceWin(int *nums, int numsSize)
{
    // just have to check if the sum of single digit number is equals to double
    // digit numbers
    int single = 0;
    int doubleDigit = 0;

    for (int i = 0; i < numsSize; i++)
    {
        if (nums[i] < 10)
        {
            single += nums[i];
        }
        else
        {
            doubleDigit += nums[i];
        }
    }

    return !(single == doubleDigit);
}

File: Java/Easy/3232.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        // just see if the sum of single digit is greater than the sum of double digit
        int singleSum = 0;
        int doubleSum = 0;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] < 10) {
                singleSum += nums[i];
            } else {
                doubleSum += nums[i];
            }
        }
        return !(singleSum == doubleSum);
    }
}